examples/custom-log-format/README.md

```diff
@@ -6,12 +6,9 @@ This example lets you set the log-format for NGINX using the configmap reosurce
 kind: ConfigMap
 apiVersion: v1
 metadata:
-  name: nginx-config
+  name: oss-nginx-ingress
 data:
-  log-format: |
-      compression '$remote_addr - $remote_user [$time_local] '
-                       '"$request" $status $bytes_sent '
-                       '"$http_referer" "$http_user_agent" "$gzip_ratio"'
+  log-format:  $remote_addr - $remote_user [$time_local] \"$request\" $status $body_bytes_sent \"$http_referer\"  \"$http_user_agent\" \"$http_x_forwarded_for\" $resource_name $resource_type $resource_namespace $service;
```

Comment:
since  `$resource_name $resource_type $resource_namespace $service` variables can be empty, better to wrap them in `"` like `"$resource_name"`. This way when logs are parsed (by some tool), it will be possible to understand which variable are empty - in that case in the logs empty variables will become empty strings enclosed in `""`

also, you can wrap the whole format in `'` so that you don't need to escape every `"`. For example:
`log-format: '$remote_addr ... "$service"'`